src/App.js

```diff
@@ -0,0 +1,29 @@
+import logo from './logo.svg';
+import './App.css';
+
+function App() {
+  return (
+    <div className="App">
+      <header className="App-header">
+        <img src={logo} className="App-logo" alt="logo" />
+        <p>
+          Edit
+          {' '}
+          <code>src/App.js</code>
+          {' '}
+          and save to reload.
+        </p>
+        <a
+          className="App-link"
+          href="https://reactjs.org"
+          target="_blank"
+          rel="noopener noreferrer"
+        >
+          Learn React
+        </a>
+      </header>
+    </div>
+  );
+}
```

Comment:
- [ ] Since we are about to delete the `logo.svg` file we can clean this component removing the boilerplate code. Then we can simply return a `Hello World` message.
```JS
function App() {
  return (
    <h1>Hello World</h1>
  );
}
```